mobile/lib/features/wallet/wallet_screen.dart

```diff
@@ -84,7 +84,9 @@ class _WalletScreenState extends State<WalletScreen> {
                       Padding(
                         padding: const EdgeInsets.only(top: 20, left: 10, right: 10, bottom: 5),
                         child: Text(
-                            "Your Payment will be shown up in the wallet history automatically once it has been processed!",
+                            PendingPaymentState.failed == state
+                                ? "Failed to submit your payment. Please make sure that you have sufficient funds that the receiver is able to receive."
```

Comment:
Is there an `and` missing?

```suggestion
                                ? "Failed to submit your payment. Please make sure that you have sufficient funds and that the receiver is able to receive."
```

Reply:
Maybe? I thought the sentence makes sense that way. You must have sufficient funds that the receiver is able to receive.

Maybe to complicated? To be honest I haven't spent much time on it, but figured that's such an easy thing ticket, let's fix it very quickly, knowing that we have to rework that entire screen anyways 🙂

Reply:
My english is not the yellow from the egg. 
I think it makes sense as you had it.